### ISSUE TITLE
2 Letter hits

### ISSUE THREAD

**Issue by ghost** [lucax88x/CodeAceJumper#6]
I'm unable to jump to 2 letter hits. So if more then 26 hits are found and the target has two letters I'M unable to type two letters.

**Comment by lucax88x** [comment]
right, I work on it right now!, thanks @con-ssc

**Comment by lucax88x** [comment]
I changed the system, instead of having double letter we have sub-groups (not an original idea from me, but inspired by the emacs one)

when we have more matches than our alphabet (a-z) we 'group' them with a sub letter, so we now have alphabet^2 size, but it's intelligent because the system will try to have more "single type" group instead of "double type" group, let's see example below which is easier to understand

let's suppose our alphabet is only A B C

if we have something like

```
function fun(){
   let text = "HAHA, you fun, funny thing!";

   return text;
}
```

and we want to Ace Jump to F, we have 4 matches.

instead of having

```
[A]unction [A]un(){
   let text = "HAHA, you [A]un, [B]unny thing!";

   return text;
}
```

which will then result in
- if I press A, I will have

```
[A]unction [B]un(){
   let text = "HAHA, you [C]un, funny thing!";

   return text;
}
```
- if I Press B I go directly to 'funny thing' as it contains only one match

our intelling system will, instead

```
[A]unction [A]un(){
   let text = "HAHA, you [B]un, [C]unny thing!";

   return text;
}
```

As you can see, first two 'A' placeholders will require another type, but the other 'B' and 'C' no.

**using only 3 alphabet characters diminishes the advantages of this system**, so with a normal alphabet this is largely emphasized, please let me know your feedbacks!

**Comment by lucax88x** [comment]
Released 1.1.0 version of the extension
